File: c2025/banking/banking/bank_main.c
//#define _CRT_SECURE_NO_WARNINGS
//#include <stdio.h>
//
//// 은행 업무 시스템
//int main() {
//	int choice = 0;   //메뉴 선택
//	int amount = 0;   //금액
//	int balance = 0;  //잔고
//
//	while (1) {
//		printf("==========================================\n");
//		printf("1.예금 | 2.출금 | 3.잔액조회 | 4.종료\n");
//		printf("==========================================\n");
//		printf("선택> ");
//		scanf("%d", &choice);
//
//		switch (choice) {
//		case 1:
//			printf("예금액> ");
//			scanf("%d", &amount);
//			balance += amount;
//			printf("%d원이 정상 입금 처리되었습니다.\n", amount);
//			break;
//		case 2:
//			printf("출금액> ");
//			scanf("%d", &amount);
//			if (amount > balance) {
//				printf("한도 초과입니다. 현재 잔액 : %d원\n", balance);
//			}
//			else {
//				balance -= amount;
//				printf("%d원이 정상 출금 처리되었습니다.\n", amount);
//			}
//			break;
//		case 3:
//			printf("잔고 : %d원\n", balance);
//			break;
//		case 4:
//			printf("프로그램을 종료합니다.");
//			return;
//		default:
//			printf("지원되지 않는 기능입니다.");
//			break;
//		}
//	}
//	
//	return 0;
//}

File: pyworks/classes/은행업무/bank_main.py

# 은행 업무
def main():
    balance = 0  #전역 변수
    while True:
        print("=========================================")
        print("1. 입금 | 2. 출금 | 3. 잔액조회 | 4. 종료 ")
        print("=========================================")
        choice = input("선택> ")
        
        if choice == '1':
            amount = int(input("입금액> "))
            balance += amount  # 잔액 = 잔액 + 입금액
            print(f"{amount}원 입금되었습니다. 현재 잔액: {balance}")
        elif choice == '2':
            amount = int(input("출금액> "))
            if amount > balance:
                print("잔액이 부족합니다.")
            balance -= amount  # 잔액 = 잔액 + 출금액
            print(f"{amount}원 출금되었습니다. 현재 잔액: {balance}")
        elif choice == '3':
            print(f"잔액> {balance}")
        elif choice == '4':
            print("프로그램을 종료합니다.")
            break
        else:
            print("지원되지 않는 기능입니다. 다시 선택해 주세요")
        
# 실행
main()  
